.circleci/config.yml

```diff
@@ -26,16 +26,38 @@ jobs:
           command: echo "export PKG=$(echo *.tar.gz)" >> $BASH_ENV
       - run:
           name: "check package"
-          command: R CMD check --no-manual --no-build-vignettes ${PKG}
+          command: |
+            R CMD check --no-manual ${PKG}
+            result=$(tail -1 rtables.Rcheck/00check.log)
+            [[ $result = Status:* ]] || echo "could not get Status line"; set -o errexit
+            [[ $result = *"Error"* ]] && echo "checks result in errors"; set -o errexit
       - run:
           name: "install package"
           command: R CMD INSTALL ${PKG}
-#      - store_artifacts:
-#          path: rtables.Rcheck
-#          destination: rtables.Rcheck
+      - store_artifacts:
```

Comment:
I think we want the install output and if possible even the build output as well as just 00check.log

Reply:
That information is available in the Steps:

![image](https://user-images.githubusercontent.com/7786462/80275183-d524a900-86df-11ea-9aa9-79c32300c063.png)

I will remodel the the pipeline to group them nicer together and I will also add the install output from the .Rcheck folder.